src/handlebars/installation.handlebars

```diff
@@ -64,21 +79,236 @@
               <p>and confirm the command output succeeds.</p>
             </div>
 
-            <div style="margin-top: 3.0em;">
-              <h2 class="bold">Migration Guide</h2>
-              <p>
-                If you're migrating to AdoptOpenJDK, you can learn about the differences between Oracle JDK and OpenJDK in our
-                <a href="./migration.html" target="_blank">Migration Guide</a> and any steps that you might need to take.
-                For example, how to use IcedTea-Web as an alternative to Webstart.
-              </p>
             </div>
+          \{{/each}}
+        </script>
+      </div>
+    </div>
+    <div>
+      <br><br>
+      <hr>
+    </div>
+
+    <h2 id="installers">Installers</h2>
+
+    <div class="margin-bottom">
+      <p>Installers are currently available for Windows&reg; and macOS&reg; JDK and JRE packages, which
+      can be obtained from the <a href="./index.html">Download page</a>. Installation steps are covered in the following sections:
+        <ul>
+          <li><a href="#windows-msi">Windows MSI installer packages</a></li>
+          <li><a href="#macos-pkg">macOS PKG installer packages</a></li>
+        </ul></p>
+
+    </div>
+
+    <div class="margin-bottom">
+
+      <h3 id="windows-msi">Windows MSI installer packages</h3>
+
+        <p>AdoptOpenJDK Windows installer packages are available as standard <tt>.msi</tt> files, which can be run with an interactive user interface or run silently
+        from the command line. The installer is designed for use on a per-machine basis, not per-user basis, so you can have only one installation of the MSI
```

Comment:
Is that still true?  I think the overall AdoptOpenJDK installer is per user (although there is a discussion for that to go back to per user)

Reply:
@karianna  this is the opposit as you said , @SueChaplain is right for all public installer available today
and for future installer ( if feature will be keep ) it will be still default per machine but optionnaly (set by a property ) per-user

Reply:
Suggest we leave this as-is and update it in the future when that work completes.